lang/en.json

```
@@ -1100,6 +1106,11 @@
         "OneTrustTagDescription": "The OneTrust Consent Management Platform orchestrates the collection of consent across web, mobile, and CTV.",
         "OneTrustTagDomainTitle": "Data-domain-script",
         "OneTrustTagDomainDescription": "To find your OneTrust data-domain-script, log into OneTrust, navigate to Scripts, select your desired domain, select the Production Scripts tab, and  find your data-domain-script in the Production CDN Script.",
-        "ContainerDashboardDescription": "With Matomo Tag Manager (MTM), you can adjust your tracking configurations without making changes to your website. The container dashboard provides an overview of tags, triggers, and variables created in the container, and a list of the most recent saved versions of the container. %1$sLearn more%2$s."
+        "ContainerDashboardDescription": "With Matomo Tag Manager (MTM), you can adjust your tracking configurations without making changes to your website. The container dashboard provides an overview of tags, triggers, and variables created in the container, and a list of the most recent saved versions of the container. %1$sLearn more%2$s.",
+        "ChooseWebsite": "Choose website",
+        "CopyContainerDescription": "Select the website to copy this container to. This will create an exact copy of all settings, tags, triggers, and variables in the chosen site.",
```

@snake14 Should we mention that it will copy the container in draft state ?
```suggestion
        "CopyContainerDescription": "Select the website to copy this container to. This will create an exact copy of all settings, tags, triggers, and variables in the chosen site in draft state.",
```

---

A good suggestion. I'm inclined to leave it for now as that's the text that Product and our Technical Writer agreed on.